be/src/olap/memtable.h

```diff
@@ -178,12 +186,18 @@ class MemTable {
     //for vectorized
     vectorized::MutableBlock _input_mutable_block;
     vectorized::MutableBlock _output_mutable_block;
+    size_t _last_sorted_pos = 0;
 
+    //return number of same keys
+    int _sort();
+    template <bool is_final>
+    void finalize_one_row(RowInBlock* row, const vectorized::ColumnsWithTypeAndName& block_data,
```

Comment:
Private method name should starts with an underscore
 
```suggestion
    void _finalize_one_row(RowInBlock* row, const vectorized::ColumnsWithTypeAndName& block_data,
```